src/components/Login/LoginFormJsonFile/LoginFormJsonFile.js

```diff
@@ -0,0 +1,65 @@
+
+import React from 'react';
+import { bool, string, func } from 'prop-types'; import { noop } from 'lodash';
+
+import ReactFileReader from 'react-file-reader'

```

Comment:
I'd prefer we not add extra dependencies that aren't needed, especially considering that we have access to electron and fs, which makes this quite simple.  You can see an example of how we're saving newly created accounts to a wallet file via the [SaveAccount component](https://github.com/nos/client/blob/master/src/components/Register/AccountDetails/SaveAccount/SaveAccount.js#L55-L75).  We should be able to inverse this logic pretty easily, I think.

Reply:
OK, changed locally.
I am now using electron to load the wallet file and fs to read the data in it instead.